frontend/app/controllers/user-home.js

```diff
@@ -1,4 +1,5 @@
 import Ember from 'ember';
+import _ from 'lodash';
```

Comment:
I had done this initially, but then there was an esllint error with using zip.appy . It asked me to use spread, and I couldn't figure out the syntax. So i reverted.